product_docs/docs/pgd/4/rel_notes/pgd_4.3.1_rel_notes.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.1"
+navTitle: "Version 4.3.1"
+---
+
+EDB Postgres Distributed version 4.3.1 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.1   | Feature         | Add pid to the log message emitted upon writer process death (BDR-2534) |
+| BDR       | 4.3.1   | Feature         | Make Raft leader step down on loss of quorum  (isolated) |
+| BDR       | 4.3.1   | Bug fix         | Fix memory leak in consensus process (BDR-3409, RT91830) |
+| BDR       | 4.3.1   | Bug fix         | Fix issue where a node can be inconsistent with the group after rejoining If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
```

Comment:
```suggestion
| BDR       | 4.3.1   | Bug fix         | Fix issue where a node can be inconsistent with the group after rejoining. If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
```